runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridConfig.java

```diff
@@ -60,4 +60,14 @@ default boolean showHighlight()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "dragDelay",
+		name = "Drag Delay",
+		description = "Number of ticks to wait after item press before showing grid"
```

Comment:
This would not make sense to players, as players will interpret "ticks" to mean server ticks, which would be 0.6 sec per tick. I'd much prefer to see this changed to be measured in ms and compare durations against that.

Reply:
I totally agree, I didn't actually realize it wasn't in server ticks myself.